src/dotnet-core-uninstall/Shared/Commands/UninstallCommandExec.cs

```diff
@@ -216,13 +216,13 @@ public static bool AskWithWarningsForRequiredBundles(IDictionary<Bundle, string>
                     LocalizableStrings.MacRequiredBundleConfirmationPromptOutputFormat, pair.Key.DisplayName, pair.Value));
                 Console.ResetColor();
                 
-                var response = userResponse?.ToUpper() ?? Console.ReadKey().KeyChar.ToString().ToUpper();
+                var response = (userResponse ?? Console.ReadLine()).ToUpper();
 
                 if (response.Equals("N"))
                 {
                     return false;
                 }
-                if (!(response.Equals("Y") || response.Equals("YES")))
+                if (!response.Equals("Y"))
```

Comment:
So, `yes` doesn't work anymore? I guess it only asks for `y/n`, so that's fine by me.

Reply:
Yes, this was briefly discussed with @joeloff https://github.com/dotnet/cli-lab/pull/331#discussion_r1974555062